src/components/Request/Pay/Views/Initial.view.tsx

```diff
@@ -33,41 +38,129 @@ export const InitialView = ({
         open()
     }
 
+    const switchNetwork = async (chainId: string) => {
```

Comment:
why do we need a new function to switch networks? 

If we need it, lets use it consistently on all pages